src/components/admin/view-sessions/AdminViewSessionCard.tsx

```diff
@@ -108,13 +109,29 @@ export function AdminViewSessionCard({
           </p>
         </div>
       </div>
-      <Button
-        className="w-full font-semibold"
-        disabled={attendees === 0}
-        onClick={handleButtonClick}
-      >
-        Download attendees list
-      </Button>
+      <div className="grid gap-2 sm:grid-cols-2">
+        <Button
+          className="w-full font-semibold"
+          disabled={attendees === 0}
+          onClick={handleButtonClick}
+        >
+          Download attendees list
+        </Button>
```

Comment:
Remove this button from the card, and all associated functions. It should just be the View Attendees List button by itself